.ci/jenkins/Jenkinsfile.e2e.cluster

```diff
@@ -181,9 +181,9 @@ void executeInCluster(Closure executeClosure) {
 void getPlatformCRFilePath() {
     switch (getClusterName()) {
         case minikubeClusterPlatform:
-            return 'config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform_withCache_minikube.yaml'
+            return 'config/samples/sonataflow.org_v1alpha08_sonataflowplatform_withCache_minikube.yaml'
         case openshiftClusterPlatform:
-            return 'config/samples/sw.kogito_v1alpha08_kogitoserverlessplatform_openshift.yaml'
+            return 'config/samples/sonataflow.org_v1alpha08_sonataflowplatform_openshift.yaml'
```

Comment:
I'm a little bit confused by what we would like to use as a `domain` and as a `group` due to the fact that we are using the `operator-sdk`.
Could we clarify it? Because if we would like to use:

- domain: org
- group: sonataflow
- 
The samples in the config folder should have only sonataflow_*
Anyway I would go with something like:

- domain: apache.org
- group: sonataflow 

or even better:

- domain: apache.org
- group: flow.sonata

Reply:
I was going to ask about it.

Reply:
sonataflow.org will be our domain. I won't add Apache there for now.

> or even better:

> group: flow.sonata

This won't match the project's name. The name of the project is Sonata Flow. Not Sonata.

Reply:
If we were to add Apache to the name, we would use apache.org as domain and group sonataflow.